base/ca/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# https://docs.fedoraproject.org/en-US/containers/guidelines/guidelines/
+
+FROM registry.fedoraproject.org/fedora:latest
+
+ARG NAME="pki-ca"
+ARG SUMMARY="Dogtag PKI Certificate Authority"
+ARG LICENSE="GPLv2 and LGPLv2"
+ARG VERSION="0"
+ARG ARCH="x86_64"
+ARG MAINTAINER="Dogtag PKI Team <devel@lists.dogtagpki.org>"
+ARG VENDOR="Dogtag"
+ARG COMPONENT="dogtag-pki"
+ARG COPR_REPO="@pki/master"
+
+LABEL name="$NAME" \
+      summary="$SUMMARY" \
+      license="$LICENSE" \
+      version="$VERSION" \
+      architecture="$ARCH" \
+      maintainer="$MAINTAINER" \
+      vendor="$VENDOR" \
+      usage="podman run -p 8080:8080 -p 8443:8443 $NAME" \
+      com.redhat.component="$COMPONENT"
+
+EXPOSE 8080 8443
+
+# Enable COPR repo if specified
+RUN if [ -n "$COPR_REPO" ]; then dnf install -y dnf-plugins-core; dnf copr enable -y $COPR_REPO; fi
+
+# Import PKI sources
+COPY . /tmp/pki/
+WORKDIR /tmp/pki
+
+# Install PKI dependencies
+RUN dnf install -y dnf-plugins-core rpm-build
```

Comment:
The dnf-plugins-core is installed twice is copr is needed and rpm-build is not in copr. It could be moved before line 33 so you can call just once.

Reply:
Good catch! I dropped the extra `dnf-plugins-core` but moved `rpm-build` after line 33 just in case we need to temporarily install something from COPR.